rasa_nlu/train.py

```diff
@@ -62,6 +62,21 @@ def init():     # pragma: no cover
     return config
 
 
+def do_train_in_worker(config):
```

Comment:
Oh sure, what I mean is that the code is pretty much the same as in this function: https://github.com/RasaHQ/rasa_nlu/pull/450/files/6905895a3216ed6fd05b4a5f228cf3a1c7e8fcae#diff-46cb37b102ce8df1048d6f487b52fc8bR80 (so why copy the lines of the function instead of calling it)